### PR TITLE
[MOB-4735] Update web sdk documentation for Safari-related bugfix

### PR DESCRIPTION
**Pull request by pauljung14**
## JIRA Ticket(s) if any

* [MOB-4735](https://iterable.atlassian.net/browse/MOB-4735)

## Description

Updates README.md for MOB-4678

## Test Steps
README renders properly in Preview.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,17 +76,19 @@ SDK Specific Options:
...
+| bottomOffset                                     | How much space (px or %) to create between the bottom of the screen and messages. Not applicable for center, top, or full-screen messages                                                                                         | `string`                                                          | `undefined` |
+| displayInterval                                  | How much time (in MS) to wait before showing next in-app message after closing the currently opened one                                                                                                                           | `number`                                                          | `30000`     |
+| handleLinks<span style="color:#EF3D55">\*</span> | How to open links. If `undefined`, use browser-default behavior. `open-all-new-tab` opens all in new tab, `open-all-same-tab` opens all in same tab, `external-new-tab` opens only off-site links in new tab, otherwise same tab. | `'open-all-new-tab' \| 'open-all-same-tab' \| 'external-new-tab'` | `undefined` |
```
Comment (pauljung14) [inline README.md:84]: This is the only line changed in this table. Added a red asterisk after handleLinks.

File: README.md [hunk 2]
```diff
@@ -76,17 +76,19 @@ SDK Specific Options:
...
+| topOffset                                        | How much space (px or %) to create between the top of the screen and messages. Not applicable for center, bottom, or full-screen messages                                                                                         | `string`                                                          | `undefined` |
+| closeButton                                      | Properties to show a custom close button on each in-app message                                                                                                                                                                   | `CloseButtonOptions` (see below)                                  | `undefined` |
+
+<span style="color:#EF3D55">*</span>Due to a limitation in Safari, web in-app messages displayed on Safari will not respect the value set for handleLinks. Customers who want to make sure links in their web in-app messages work in Safari will need to specify the **target** property in their message (`target="_blank"` for new tab, `target="_top"` / `target="_parent"` for same tab).
```
Comment (pauljung14) [inline README.md:91]: Note refers back to handleLinks in table via red asterisk.

File: README.md [hunk 3]
```diff
@@ -76,17 +76,19 @@ SDK Specific Options:
...
+| bottomOffset                                     | How much space (px or %) to create between the bottom of the screen and messages. Not applicable for center, top, or full-screen messages                                                                                         | `string`                                                          | `undefined` |
+| displayInterval                                  | How much time (in MS) to wait before showing next in-app message after closing the currently opened one                                                                                                                           | `number`                                                          | `30000`     |
+| handleLinks :rotating_light: | How to open links. If `undefined`, use browser-default behavior. `open-all-new-tab` opens all in new tab, `open-all-same-tab` opens all in same tab, `external-new-tab` opens only off-site links in new tab, otherwise same tab. | `'open-all-new-tab' \| 'open-all-same-tab' \| 'external-new-tab'` | `undefined` |
```
Comment (pauljung14) [inline README.md:84]: This is the only line changed in this table. Added a rotating light icon after handleLinks. The other rows changed in this table are formatting changes.

File: README.md [hunk 4]
```diff
@@ -76,17 +76,19 @@ SDK Specific Options:
...
+| topOffset                                        | How much space (px or %) to create between the top of the screen and messages. Not applicable for center, bottom, or full-screen messages                                                                                         | `string`                                                          | `undefined` |
+| closeButton                                      | Properties to show a custom close button on each in-app message                                                                                                                                                                   | `CloseButtonOptions` (see below)                                  | `undefined` |
+
+:rotating_light: Due to a limitation in Safari, web in-app messages displayed on Safari will not respect the value set for handleLinks. Customers who want to make sure links in their web in-app messages work in Safari will need to specify the **target** property in their message (`target="_blank"` for new tab, `target="_top"` / `target="_parent"` for same tab).
```
Comment (pauljung14) [inline README.md:91]: Note refers back to handleLinks in table via red rotating light.

**Review by mprew97 — APPROVED** [review]

**Pull request merged**